contracts/src/CrossChainApplications/ERC20Bridge/IERC20Bridge.sol

```diff
@@ -11,14 +11,14 @@ import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
  * @dev Interface that describes functionalities for a cross-chain ERC20 bridge.
  */
 interface IERC20Bridge {
-    /**
-     * @dev Enum representing the action to take on receiving a Teleporter message.
-     */
+
     enum BridgeAction {
         Create,
         Mint,
         Transfer
-    }
+    }    /**
```

Comment:
Why was the comment moved below the enum? Seems to go against our coding convention.

Reply:
Whoops, copypasta